pkg/loadbalancer/reconciler/termination.go

```diff
@@ -180,17 +182,22 @@ func terminateUDPConnectionsToBackend(p socketTerminationParams, l3n4Addr lb.L3n
 	l4Addr := l3n4Addr.L4Addr
 
 	switch l3n4Addr.Protocol {
-	case lb.UDP, lb.ANY:
```

Comment:
Do we still have the flag to disable the protocol differentiation? I thought when it was turned off, everything came in as ANY.